src/components/cart/CartItem/cart-item.module.scss

```diff
@@ -0,0 +1,74 @@
+@import "src/styles/scaffold";
+
+.fs-cart-item {
+  // --------------------------------------------------------
+  // Design Tokens for CartItem
+  // --------------------------------------------------------
+
+  // Default properties
+  --fs-cart-item-padding               : var(--fs-spacing-3);
+  --fs-cart-item-bkg-color             : var(--fs-control-bkg);
+
+  --fs-cart-item-border-width          : var(--fs-border-width);
+  --fs-cart-item-border-color          : var(--fs-border-color-light);
+  --fs-cart-item-border-radius         : var(--fs-border-radius);
+
+  // Image
+  --fs-cart-item-image-border-radius   : var(--fs-cart-item-border-radius);
+
+  // Title
+  --fs-cart-item-title-margin-bottom   : var(--fs-spacing-0);
+  --fs-cart-item-title-line-height     : 1.2;
+  --fs-cart-item-title-color           : var(--fs-color-text);
+
+  // Actions
+  --fs-cart-item-actions-margin-top    : var(--fs-spacing-3);
+
+  // --------------------------------------------------------
+  // Structural Styles
+  // --------------------------------------------------------
+
+  padding: var(--fs-cart-item-padding);
+  background-color: var(--fs-cart-item-bkg-color);
+  border: var(--fs-cart-item-border-width) solid var(--fs-cart-item-border-color);
+  border-radius: var(--fs-cart-item-border-radius);
+
+  [data-card-content] {
```

Comment:
Noticed on [the NextJS version](https://github.com/vtex-sites/nextjs.store/commit/272138f8284ca3293b3a67b6c324384d7ab5730a#diff-ca05f1a8a5a0e700813d8ace79209b049f0865d3e9ca2e1a16fdb00e5eca85f2R36) you had added more changes from `data-fs-cart-item-*`, but not here. Are you planning to that change later? With all the syncing effort going on, I think it may be important to keep them equal as much as possible.

![CleanShot 2022-06-07 at 19 39 49](https://user-images.githubusercontent.com/381395/172494948-2306ea1d-0ece-4f83-bc50-e22cc3269922.png)

Reply:
Well noticed! @filipewl 

I miss cherry-picking [this commit](https://github.com/vtex-sites/nextjs.store/pull/102/commits/aa71acd163e612425b10dbf54edddef71a5f4e0d). Thank you!